public/components/FeatureAnywhereContextMenu/AddAlertingMonitor/CreateNew/CreateNew.tsx

```diff
@@ -0,0 +1,227 @@
+    VisualizeEmbeddable | ErrorEmbeddable
+  >();
+  const isAd = values.searchType === SEARCH_TYPE.AD;
+  const [accordionsOpen, setAccordionsOpen] = useState(detectorId ? { monitorDetails: true } : {});
+  const [isShowVis, setIsShowVis] = useState(false);
+  const title = embeddable.vis.title;
+  const notificationService = useMemo(() => new NotificationService(getClient()), []);
+
+  const onAccordionToggle = (key) => {
+    const newAccordionsOpen = { ...accordionsOpen };
+    newAccordionsOpen[key] = !accordionsOpen[key];
+    setAccordionsOpen(newAccordionsOpen);
+  };
+  const { isShowingErrors, errorList } = useMemo(() => {
+    const isShowingErrors = Object.keys(errors).length > 0 && !isSubmitting && !isValid;
+    const errorList = new Set();
+
+    if (isShowingErrors) {
+      const errorPaths = dotNotate(errors);
+
+      Object.keys(errorPaths).forEach((path) => {
+        errorList.add(errorPaths[path]);
+      });
+      // Set accordion values if there are errors for top level
+      const newAccordionsOpen = { ...accordionsOpen };
+      if (!newAccordionsOpen.advancedData)
+        newAccordionsOpen.advancedData = 'aggregations' in errors || 'groupBy' in errors || 'where' in errors || 'bucketValue' in errors;
+      if (!newAccordionsOpen.monitorDetails)
+        newAccordionsOpen.monitorDetails = 'name' in errors || 'period' in errors;
+      setAccordionsOpen(newAccordionsOpen);
+    }
+
+    return { isShowingErrors: errorList.size !== 0, errorList: [...errorList] };
+  }, [errors, isSubmitting, isValid]);
+
+  // On load
+  useEffect(() => {
+    const updatePlugins = async () => {
+      const newPlugins = await getPlugins(httpClient, getEmbeddable());
+      setPlugins(newPlugins);
+    };
+
+    const createEmbeddable = async () => {
+      const visEmbeddable = await fetchVisEmbeddable(embeddable.vis.id, getEmbeddable(), getQueryService());
+      setGeneratedEmbeddable(visEmbeddable);
+    };
+
+    updatePlugins();
+
+    createEmbeddable();
+  }, []);
+
+  return (
+    <div className="create-new">
+      <EuiText size="xs">
+        <p>
+          {flyoutMode === 'create' &&
+            'Create query level monitor, associated with the visualization. Learn more in the documentation.'}
+          {flyoutMode === 'adMonitor' &&
+            'Set up and configure alerting monitor for the anomaly detector to receive notifications on visualization when anomalies detected.'}{' '}
+          <a
+            href="https://opensearch.org/docs/latest/monitoring-plugins/alerting/index/"
+            target="_blank"
+          >
+            Learn more <EuiIcon type="popout" />
+          </a>
+        </p>
+      </EuiText>
+      {isShowingErrors && showErrors && (
+        <>
+          <EuiSpacer size="m" />
+          <EuiCallOut title="Address the following errors:" color="danger" iconType="alert">
+            <ul>
+              {errorList.map((text) => (
+                <li key={text}>{text}</li>
```

Comment:
for `key={text}`, the text might be too big, let's use the index from map instead?
```
errorList.map((text, index) => <li key={index} ...
```